### DISCUSSION TITLE
Couple of issues using Leantime

### DISCUSSION THREAD

**Discussion by JorgeCarousel** [Leantime/leantime#672] [Q&A]
Hey guys,

I installed Leantime as a test run, but I've been having a couple of issues, how can I fix these?:

1. I created a couple of users but when I try to edit any of them I get a blank page.
2. I installed Leantime in a folder called Leantime on my web server so the URL would be something like this: https://server.example.com/Leantime but for some reason in the menu link for To-Dos shows as being: https://server.example.com/Leantime/Leantime/tickets/showKanban (Leantime twice) No other link shows with this second Leantime on it.

Thanks.

**Comment by marcelfolaron** [comment]
1. Is fixed in 2.1.9
2. Please set the $appURL in your config file to be https://server.example.com/Leantime

**Comment by JorgeCarousel** [comment]
Thank you Marcel, much appreciated.





=============

Jorge F. Hernandez

Systems Administrator

Carousel



*From:* Marcel Folaron ***@***.***>
*Sent:* Friday, April 22, 2022 10:07 AM
*To:* Leantime/leantime ***@***.***>
*Cc:* JorgeCarousel ***@***.***>; Author ***@***.***>
*Subject:* Re: [Leantime/leantime] Couple of issues using Leantime
(Discussion #672)



   1. Is fixed in 2.1.9
   2. Please set the $appURL in your config file to be
   https://server.example.com/Leantime

—
Reply to this email directly, view it on GitHub
<https://github.com/Leantime/leantime/discussions/672#discussioncomment-2615925>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AYS4IAEGGWXAUHPBC7UP7TDVGKW7HANCNFSM5SWX56RA>
.
You are receiving this because you authored the thread.Message ID: <
***@***.***>
